custom_components/waste_collection_schedule/waste_collection_schedule/source/nvaa_se.py

```diff
@@ -0,0 +1,121 @@
+from datetime import datetime
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Norrtalje Vatten & Avfall"
+DESCRIPTION = "Source for Norrtalje Vatten & Avfall waste collection services, Sweden."
+URL = "https://sjalvservice.nvaa.se/api/v1/integration/getNextGarbageCollection"
```

Comment:
This parameter will is shown in the README.md and should not be the API URL but the domain of the service provider (probably https://sjalvservice.nvaa.se/?), you can rename this parameter to something like API_URL for use within this script

Reply:
Now fixed